src/gui/properties/propertieswidget.cpp

```diff
@@ -143,7 +143,8 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
     m_ui->vBoxLayoutPeerPage->addWidget(m_peerList);
     // Tab bar
     m_tabBar = new PropTabBar(nullptr);
-    m_tabBar->setContentsMargins(0, 5, 0, 0);
+    m_tabBar->setContentsMargins(0, 3, 0, 3);
+    m_ui->verticalLayout->setSpacing(0);
```

Comment:
I don't think this line do anything... or would you point out please?

Reply:
![image](https://user-images.githubusercontent.com/34717789/70391491-e48aaa80-19fb-11ea-8209-37db76f1a6b9.png)

Reply:
without this spacing is 6, so that will add extra top margin.

Reply:
Well I see the effect now but I can't say I like it.
Try open & close the Trackers tab, you'll notice the height of the red block shrink & expand.

![pic](https://user-images.githubusercontent.com/9395168/70393661-52989700-1a27-11ea-815c-3c9acd710cac.png)
And btw this should be modified at the .ui file.

Reply:
hmm but why🤔? I will get back it after some time.

Reply:
Do you agree with a vertically centered proptab bar? I mean the same top and bottom spacing. I mean you get it XD.

Reply:
This actually happens with official releases as well.

Reply:
>Do you agree with a vertically centered proptab bar? I mean the same top and bottom spacing.

I don't object it, however IIRC it is hard to ensure that on every desktop environment. You'll also need to test with 1 or 2 desktop environment on linux besides windows.

Reply:
I think extra space is related to #11622

Reply:
@Chocobo1  can you take a fresh look. This also fixes #11622.
![image](https://user-images.githubusercontent.com/34717789/70413897-8fdc4380-1a7e-11ea-9b24-362cfd69c817.png)
![image](https://user-images.githubusercontent.com/34717789/70413913-9c609c00-1a7e-11ea-892c-481c27a41e20.png)